### PR TITLE
[Chore](case) add some regression-test case about materialized-view

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
# Proposed changes

add some regression-test case about materialized-view

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

File: regression-test/suites/mv_p0/sum_divede_count/sum_devide_count.groovy [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                k4 varchar(100) null
+            )
+            duplicate key (k1,k2,k3)
+            distributed BY hash(k1) buckets 3
+            properties("replication_num" = "1");
+        """
+
+    sql "insert into d_table select 1,1,1,'a';"
+    sql "insert into d_table select 2,2,2,'b';"
+    sql "insert into d_table select 3,-3,null,'c';"
```
Comment (luozenglin) [inline regression-test/suites/mv_p0/sum_divede_count/sum_devide_count.groovy:37]: need `sync`?

Comment (yiguolei) [reply regression-test/suites/mv_p0/sum_divede_count/sum_devide_count.groovy:37]: not need sync here, because the following sql is create mv

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**